examples/templating-7.10.dockerfile

```diff
@@ -4,4 +4,4 @@ RUN cabal update
 ADD mypackage.cabal /app/mypackage.cabal
 RUN cabal install --only-dep -j
 ADD . /app/
-RUN cabal build
+RUN cabal build
```

Comment:
Is there a reason for removing *newlines* from example files?

Reply:
I think that was done by running the examples themselves. Looks like the new script is not adding a newline. Do you think we should add it by default?

Reply:
I'll actually go ahead and do that, I think preserving the previous behaviour (automatically adding a new line) is important